examples/patterns/pagination.html

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: Pagination
+category: _patterns
+---
+
+<footer class="p-pagination">
+  <a class="p-pagination__link" href="#previous">
+    <div class="p-pagination__text">
+      <span class="p-pagination__label">Previous</span>
+      <span class="p-pagination__title">Lorem ipsum dolor sit amet</span>
+    </div>
+  </a>
+  <a class="p-pagination__link" href="#next">
+    <span class="p-pagination__label">Next</span>
```

Comment:
Is this one missing: `<div class="p-pagination__text">`?

Reply:
No, that wrap is required on the first link becuase the text is `display: none` on small screens.

Reply:
Don't you think it would be nice to have them the same to symmetry?